### PR TITLE
Feature/add models

### PR DESCRIPTION
**Pull request by taraskonchak**

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+    expiration_date = models.DateField()
+    amount = models.FloatField()
+    credit_limit = models.FloatField(null=True)
+    deposit_amount = models.FloatField(null=True)
+    credit_percentage = models.FloatField()
+    deposit_percentage = models.FloatField()
+    credit_start_date = models.DateTimeField()
+    credit_end_date = models.DateTimeField()
+    deposit_start_date = models.DateTimeField()
+    deposit_end_date = models.DateTimeField()
```
Comment (olyapetryshyn) [inline accounts/models.py]: add null=True constraint to this and 5 other upper fields

File: users/models.py [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+    phone_number = models.CharField(max_length=13, unique=True)
+    passport_number = models.CharField(max_length=8, unique=True)
+    identity_code = models.CharField(max_length=15, unique=True)
+    company = models.ForeignKey("self", on_delete=models.DO_NOTHING)
+    salary = models.FloatField()
+    birth_date = models.DateField()
+    residence_permit = models.TextField(max_length=255)
+
+
+class Roles(TimeStampedModel):
```
Comment (olyapetryshyn) [inline users/models.py]: single form(Role)

File: users/models.py [hunk 3]
```diff
@@ -1,3 +1,22 @@
+from django.contrib.auth.base_user import AbstractBaseUser
 from django.db import models
 
-# Create your models here.
+from common.models import TimeStampedModel
+
+
+class Users(TimeStampedModel, AbstractBaseUser):
```
Comment (olyapetryshyn) [inline users/models.py]: User

File: users/models.py (hunk 2 above)
Comment (taraskonchak) [reply users/models.py]: Does it really should be "Role"  instead "Roles" as in ER diagram?

Comment (taraskonchak) [reply users/models.py]: > single form(Role)

Does it really should be "Role"  instead "Roles" as in ER diagram?

File: users/models.py (hunk 3 above)
Comment (taraskonchak) [reply users/models.py]: > User

Does it really should be "User"  instead "Users" as in ER diagram?

File: accounts/models.py (hunk 1 above)
Comment (taraskonchak) [reply accounts/models.py]: > add null=True constraint to this and 5 other upper fields

Added constraints

**Review by LSikyr — DISMISSED** [review]

**Review by LSikyr — DISMISSED** [review]

File: accounts/models.py [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+    credit_start_date = models.DateTimeField(null=True)
+    credit_end_date = models.DateTimeField(null=True)
+    deposit_start_date = models.DateTimeField(null=True)
+    deposit_end_date = models.DateTimeField(null=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+
+class Transaction(TimeStampedModel):
+    account = models.ForeignKey(Account, on_delete=models.CASCADE)
+    type_of_expence = models.CharField(max_length=50)
```
Comment (olyapetryshyn) [inline accounts/models.py]: maybe we should add choices for this field

Comment (ement06) [reply accounts/models.py]: > maybe we should add choices for this field

what do you mean, could you please explain more?

File: requirements/requirements.in [hunk 5]
```diff
@@ -1 +1,2 @@
 Django
+psycopg2==2.7.7
```
Comment (ement06) [inline requirements/requirements.in]: please, add an empty row at the end of file

**Review by ement06 — COMMENTED** [review]
I want to wait for the answer from Olya on her comment

File: requirements/requirements.in (hunk 5 above)
Comment (taraskonchak) [reply requirements/requirements.in:2]: @olyapetryshyn

File: accounts/models.py (hunk 4 above)
Comment (taraskonchak) [reply accounts/models.py]: @olyapetryshyn, can you provide examples?

Comment (olyapetryshyn) [reply accounts/models.py]: https://docs.djangoproject.com/en/3.0/ref/models/fields/

Comment (olyapetryshyn) [reply accounts/models.py]: read about enumeration types

Comment (ement06) [reply accounts/models.py]: > read about enumeration types

I got it, but which choices you want to see here. We didn't speak about that.

**Review by Sikyrynskyi — APPROVED** [review]

**Review by DeepSeekyrynskyi — APPROVED** [review]

**Review by LSikyr — APPROVED** [review]

**Pull request merged**